python_basics_writing_python_code/python_basics_writing_python_code.md

```diff
@@ -196,7 +128,7 @@ print("Hello World!".lower())
 </lia-keep>
 </div>
 
-As you might have suspected, the `.lower()` method made all of the letters of the phrase "Hello World!" lowercase. The `.lower()` method is a **string method**, or a method that must be called on a string, of which "Hello World!" is an example. (There is also a string method called `.upper()`, can you guess what it does?) The Python documentation has [more information about string methods](https://docs.python.org/2.5/lib/string-methods.html).
+As you might have suspected, the `.lower()` method made all of the letters of the phrase "Hello World!" lowercase. The `.lower()` method is a **string method**, or a method that *must* be called on a string object, of which "Hello World!" is an example. (There is also a string method called `.upper()`, can you guess what it does?) The Python documentation has [more information about string methods](https://docs.python.org/2.5/lib/string-methods.html). Other kinds of objects in Python (like lists and dictionaries, for example, which we'll discuss later in this module) have their own associated methods.
 
```

Comment:
Right now the first time a learner would be seeing a method taking a parameter is in the quiz for this section. I really like `.endswith()` question and don't think you should use a different method, but maybe you could add an explanation here "functions take arguments but methods also take parameters." 

I suppose it could also go in the answer to the quiz question, but I think this might be a better spot.

Reply:
Oooh yeah that's a good catch, I'll add something here.

Reply:
Okay, I did that thing!